### ISSUE TITLE
Displaying targets or boosters to affiliates

### ISSUE THREAD

**Issue by palepaul** [anujtenani/goaffpro#216]
A few merchants have requested for an option to display targets and commission boosters in the affiliate dashboard (and possibly showing the affiliate what they need to do, to reach that target or booster)

**Comment by ProAthletesHemp** [comment]
This would be beneficial. In the mean time any comment or note that we could add on the dashboard would help.

**Comment by copperandrose** [comment]
Yes, please this is so important for sales incentive also for affiliates to know their goals, and how much they have sold. Especially if they have sub affiliates. A lot of leaders like to see their downlines so they can congratulate them too.
